src/ImageDistances.jl

```diff
@@ -14,21 +14,28 @@ import Distances:
     colwise!,
     pairwise
 
+# TODO: remove dependency on FixedPointNumbers and Colors
+# https://github.com/JuliaImages/Images.jl/issues/802
 using FixedPointNumbers
 using Colors
 using FixedPointNumbers: floattype
 using ImageCore, ColorVectorSpace
 
+
+# These traits are already defined in ImageCore
+# copied them here for compatibility consideration
```

Comment:
A better way is to (1) put a minimum bound on the ImageCore version in `Project.toml`, and then (2) say `using ImageCore: NumberLike, RealLike, FractionalLike, ...`. That way you're guaranteed to always stay in-sync with the definitions in ImageCore.

Reply:
Yes, exactly!

The reason for doing this is because some other packages haven't upgraded to the new registrator, which makes them incompatible to `ImageCore 0.8` (e.g., `GLMakie`
https://github.com/JuliaImages/ImageDistances.jl/issues/20)

I did some compatibility tradeoff here and planned to do what you said when ImageDistances is ready for v0.3.0.